packages/theming/theme-tokens/src/getMacOSAliasTokens.ts

```diff
@@ -0,0 +1,26 @@
+import * as macOSLightAliasTokens from './light-macos/tokens-alias';
+import * as macOSDarkAliasTokens from './dark-macos/tokens-alias';
+import * as macOSLightHCAliasTokens from './light-high-contrast-macos/tokens-alias';
+import * as macOSDarkHCAliasTokens from './dark-high-contrast-macos/tokens-alias';
```

Comment:
small nit: we could do `import macOSDarkHCAliasTokens  from './dark-high-contrast-macos/tokens-alias'` here instead of importing everything as macOSDarkHCAliasTokens. Then when you access macOSDarkHCAliasTokens you don't need the ".default" part.